docs/ref/applications.txt

```diff
@@ -509,3 +519,31 @@ Here are some common problems that you may encounter during initialization:
   :setting:`INSTALLED_APPS` to contain
   ``'django.contrib.admin.apps.SimpleAdminConfig'`` instead of
   ``'django.contrib.admin'``.
+
+* ``RuntimeWarning: Accessing the database during app initialization is
+  discouraged.`` This happens if one of your apps accesses the databases in the
+  :meth:`~AppConfig.ready` method or directly at import time. This is bad
+  because it will happen during the startup of every management command. For
+  example, even though the test database configuration is separate from the
+  production settings, ``manage.py test`` would still execute some queries
+  against your **production** database!
```

Comment:
Not a useful/common example? 🤔

Instead of focusing on one speculative example, consider describing the problem generally. For eg:

```
Database access during app initalisation or import time is undesirable for a number of reasons and is never required in Django projects.

For example a common mistake may be to populate the choices on a form field …
```

Reply:
Fair enough. I just copied from the warning note in the `AppConfig.ready()` section 😅